src/domain/repositories/ReportExportRepository.ts

```diff
@@ -0,0 +1,12 @@
+import { Async } from "domain/entities/Async";
+import { TrackedEntity } from "domain/entities/TrackedEntity";
+
+export interface ReportExportRepository {
+    save(options: ReportExportSaveOptions): Async<void>;
```

Comment:
[clean arch] [minor] A repository is typically paired with an entity in the domain. So the interface `ReportExportSaveOptions` can be converted to some entity in `src/domain/entities`. For example: `EntitiesReport` (the prefix because it holds tracked entities, not because it's an entity). And what to do about outputPath, which makes no sense in a entity? This field can be a generic "name".